Makefile

```diff
@@ -35,9 +35,14 @@ enabledebug:
 ps:
 	docker-compose ps
 
-test: composer web
+test: unit-tests functional-tests
```

Comment:
It looks like this lost the `composer` entry that will ensure composer dependencies are installed. Was that intentional? Just checking.

Reply:
It's still a dependency on the functional-tests entry

Reply:
Do the unit tests not need the composer dependencies? (slightly surprised...)